openpype/lib/vendor_bin_utils.py

```diff
@@ -321,7 +354,29 @@ def get_oiio_tools_path(tool="oiiotool"):
     return tool_executable_path
 
 
-def _ffmpeg_executable_validation(filepath):
+def get_oiio_tools_args(tool_name="oiiotool"):
+    """Arguments to launch OpenImageIO tool.
+
+    Args:
+        tool_name (str): Tool name 'oiiotool', 'maketx', etc.
+            Default is "oiiotool".
+
+    Returns:
+        list[str]: List of arguments.
+    """
+
+    if AYON_SERVER_ENABLED:
+        args = _get_ayon_oiio_tool_args(tool_name)
+        if args:
+            return args
+
+    path = get_oiio_tools_path(tool_name)
+    if path:
+        return [path]
+    return []
```

Comment:
I am wondering if we should allow this. All the places that are using it will actually fail if this is empty, so I am not sure if we shouldn't rise exception here or handle it in the place where this is used.